assets/js/ajax-file-upload.js

```diff
@@ -5,15 +5,39 @@ jQuery(function($) {
 			dataType: 'json',
 			dropZone: $(this),
 			url: job_manager_ajax_file_upload.ajax_url.toString().replace( '%%endpoint%%', 'upload_file' ),
-			maxNumberOfFiles: 1,
 			formData: {
 				script: true
 			},
+			change: function() {
+				this.validation_errors = [];
+			},
 			add: function (e, data) {
-				var $file_field     = $( this );
-				var $form           = $file_field.closest( 'form' );
-				var $uploaded_files = $file_field.parent().find('.job-manager-uploaded-files');
-				var uploadErrors    = [];
+				var $file_field      = $( this );
+				var $form            = $file_field.closest( 'form' );
+				var $uploaded_files  = $file_field.parent().find('.job-manager-uploaded-files');
+				var uploadErrors     = [];
+				var fileLimitLeft    = false;
+				var fileLimit        = parseInt( $file_field.data( 'file_limit' ), 10 );
+
+				if ( typeof $file_field.data( 'file_limit_left' ) !== 'undefined' ) {
+					fileLimitLeft = parseInt( $file_field.data( 'file_limit_left' ), 10 );
+				} else if ( typeof fileLimit !== 'undefined' ) {
+					var currentFiles = parseInt( $uploaded_files.children( '.job-manager-uploaded-file' ).length, 10);
+					fileLimitLeft = fileLimit - currentFiles;
+					$file_field.data( 'file_limit_left', fileLimitLeft );
+				}
+
+				if ( false !== fileLimitLeft && fileLimitLeft <= 0 ) {
+					var message = 'Exceeded upload limit';
+					if( $file_field.data( 'file_limit_message' ) ) {
+						message = $file_field.data( 'file_limit_message' );
+					} else if ( typeof job_manager_job_submission !== 'undefined' ) {
+						message = job_manager_job_submission.i18n_over_upload_limit;
```

Comment:
Should we use JS i18n here? https://make.wordpress.org/core/2018/11/09/new-javascript-i18n-support-in-wordpress/

I guess it was introduced in 5.0, so maybe we should hold off until we drop 4.x support?

Reply:
Yeah, I think let's hold off on that now. I'm a bit hesitant to bump minimum version to 5.0 with so many on 4.9.